### ISSUE TITLE
Add the WCAG S.C name and Level

### ISSUE THREAD

**Issue by ferBonnin** [microsoft/accessibility-insights-web#2565]
## Is your feature request related to a problem? Please describe.

The level of a particular failure is used for certain teams / companies to decide the priority in which issues get fixed first. Having this information surfaced in the UI and reports will make this much easier.

## Describe the desired outcome

- In FastPass >automated checks and the report the 'resources for this rule' information contain not only the S.C number but the level and name too. e.g. 'WCAG 1.4.3 Contrast (Minimum) (Level AA)'

- in Assessment, add to the report the WCAG name and level.

## Describe alternatives you've considered

Currently, the only way to do this is to select the link and look at the level but this is time consuming; would be easier to have it all as part of the tool.

## Additional context

This is a feature request filed based on a user request

**Comment by markreay** [comment]
I think the features seems like something we should do. I am find with it if we can work out the visual design.

**Comment by ferBonnin** [comment]
taking a look at this during hackathon

**Comment by ferBonnin** [comment]
If we wanted to do this without a feature flag or setting, we will just need to change _guidance-links.tsx_ (to remove a `toUpper()`) and link.tsx (to add the WCAG Name and level as part of the list of files).

Changing this would impact: automated checks details view, dialog, bug filing and report as well as the assessment guidance (Do's and Don'ts) and report. 

![automated checks with longer URL in rule resources](https://user-images.githubusercontent.com/9062104/88722692-87537e80-d0dc-11ea-8aec-5b4678698941.png)

![assessment report with longer URL](https://user-images.githubusercontent.com/9062104/88722744-9d613f00-d0dc-11ea-9887-fe2823243b5b.png)


Looked into adding a Feature flag so we can check in renderLink in _guidance-links.tsx_ (per quick conversation with @waabid) whether to show the long name or just the WCAG number but after reviewing with @karanbirsingh looks like that would involve passing the `featureFlagStoreData` into a lot of files' properties.

**Comment by ferBonnin** [comment]
Some more things to consider after a conversation with Karan:
- if we enable this with a feature flag, how do users of the report package specify preferences?
- Do & Don't content needs to change too
- Should this be a Setting, Feature Flag, or just change the content with the additional information for all users?

**Comment by jenniaynlynn** [comment]
I love this idea!  Our testing team independently came up with this idea and brought it up to ferBonnin...only to learn it was already a suggested feature. The current link text of only the "WCAG x.x.x" criterion implies existing knowledge that may not be there for people.  Even now as an SME, I can recall names of rules better than numbers and they stick with me better.  In addition, there could be many people out there setting Level A compliance as a starting point and having this information helps them prioritize the work.  

In summary - I think this fix would help more people make meaningful connections with the criteria to understand them better without having to click deeper or be an expert.

**Comment by april-ux** [comment]
Hi, team. Any updates on this? It would still be a big help.

**Comment by DaveTryon** [comment]
We have a feature on our backlog to implement WCAG 2.2 support once the spec is finalized. Closing this to avoid duplication.

Thanks for using Accessibility Insights!
